src/lsp/sql_parser/index.mld

```diff
@@ -0,0 +1,10 @@
+{1 Library sql_parser}
+
+Superbol is a set of utilities for COBOL developers,
+that can be selected by sub-commands.
+
+This package contains all the parsing logic and grammar definitions, 
```

Comment:
I think there's a field in `package.toml` where you should put this description, so `drom project` won't complain about this `index.mld` file being modified.

(Same in `sql_ast/package.toml`)